### PR TITLE
Refactor json value extraction

### PR DESCRIPTION
**Pull request by tamirkamara**
## Description
Refactor how we extract the values from the definition json and use in Terraform. 

### Changes
- Use a template for the tfvars file that includes the json paths for the values
- Have a short bash script in the workflow that populates the template file with the trial specific values
- Minimize the vars we build into the run matrix
- Delete current content in the $web ui container

This action show a successful run with 2 trials being updated:
https://github.com/NDPH-ARTS/mts-trial-deployment-config/actions/runs/574486407

### Checklist:

- [ ] Branch name follows convention (feature/arts-###-short-name OR fix/arts-###-short-name)
- [x] I have included a short-meaningful title, description and changes for this PR

### CODE REVIEW

**Review by guybartal — DISMISSED** [review]

File: tamir3/definition.json [hunk 1]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (eladiw) [inline tamir3/definition.json]: file should not be part of this pr

**Review by eladiw — APPROVED** [review]
LGTM

**Review by guybartal — APPROVED** [review]
